src/Device/Driver/XCVario.cpp

```diff
@@ -32,8 +32,13 @@ Copyright_License {
 #include "util/Clamp.hpp"
 #include "Atmosphere/Pressure.hpp"
 #include "LogFile.hpp"
+#include "Operation/Operation.hpp"
```

Comment:
Why include this header?

Reply:
The Timer callback needs a null environment, in order to declare this.

Reply:
This is bad, because it means that `OnSysTicker()` can block, but it must not. By abusing a `NullOperationEnvironment`, you escape the protection against this mistake I built into the API.